tests/platform_tests/link_flap/test_cont_link_flap.py

```diff
@@ -0,0 +1,132 @@
+"""
+Tests the continuous link flap in SONiC.
+
+Parameters:
+    --orch_cpu_threshold <port> (int): Which port you want the test to send traffic
+        to. Default is 3.
+"""
+
+import logging
+import time
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common import port_toggle
+from tests.platform_tests.link_flap.link_flap_utils import build_test_candidates, toggle_one_link, check_orch_cpu_utilization
+from tests.common.utilities import wait_until
+
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer
```

Comment:
A topo marker must be assigned, or the case will be skipped.

Reply:
Done. Thanks